controllers/activitiesController.js

```diff
@@ -1,3 +1,23 @@
-class ActivitiesController{}
+const {Activities} = require('../models')
+
+class ActivitiesController {
+
+    static async updateActivitie(req, res, next) {
+        const { id } = req.params;
+        const  body  = req.body;
+        try {
+          const updateActivitie = await Activities.findByPk(id);
+          if(updateActivitie)      
+            {
+              await updateActivitie.update(body)
+              res.status(200).json(body)
+            }else {
+              res.status(404).json({ msg: 'Could not find Activitie' });
+            }
```

Comment:
Pienso que se debería cambiar la sentencia if-else a uso de solo if
de la siguiente manera: 
await updateActivitie.update(body)
if(!updateActivitie) return res.status(404).json({ msg: 'Could not find Activitie' });
return res.status(200).json(body)
se reduce la cantidad de lineas y queda extensible a más validaciones, aunque en este caso no creo que sea necesario